examples/megacli.cpp

```diff
@@ -3316,6 +3316,40 @@ void exec_cp(autocomplete::ACState& s)
 
 void exec_du(autocomplete::ACState& s)
 {
+    if (s.extractflag("-all"))
+    {
+        bool breakdown = s.extractflag("-breakdown");
```

Comment:
breakdown should be const

Reply:
I don't think this one needs it either.    `const` is important for member functions, parameters etc, but not for local variables.

Reply:
completely disagree and so do the cpp core guidelines ;)
const should be the default in C++ but unfortunately it's not and we should add it everywhere we can to make intent clear and have the compiler help you.